apps/studio/components/interfaces/HomeNew/CustomReportSection.tsx

```diff
@@ -298,14 +298,18 @@ export function CustomReportSection() {
         {(() => {
           if (layout.length === 0) {
             return (
-              <div className="flex min-h-[270px] items-center justify-center rounded border-2 border-dashed p-16 border-default">
+              <div className="h-64 flex flex-col items-center justify-center rounded border-2 border-dashed p-16 border-default">
+                <h4>Build a custom report</h4>
```

Comment:
wondering if stating that these are from SQL snippets might be better?

"Build a custom report with SQL"

Reply:
Feedback - reckon we could add a CTA at the bottom here to "Create your first block with SQL" which directs them to the SQL editor

<img width="390" height="199" alt="image" src="https://github.com/user-attachments/assets/2b8e77d3-895f-4987-85bb-9d70a658eb4b" />

Also feedback - at the top of my head wondering if it might make sense to introduce a category of SQL snippet templates for building custom reports (understandably the needs of all projects will be different, but perhaps just the rough building blocks might be useful - e.g templates with SQL aggregation of data)

Reply:
[Edit] Added the CTA at the bottom
<img width="373" height="360" alt="image" src="https://github.com/user-attachments/assets/e9427184-84e3-4011-87ce-8d3b373693fa" />